PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/intellij/ui/SubscriptionsDialog.java

```diff
@@ -170,7 +170,7 @@ public void actionPerformed(AnActionEvent anActionEvent) {
                 model.fireTableDataChanged();
                 table.getEmptyText().setText("Refreshing");
                 AppInsightsClient.createByType(AppInsightsClient.EventType.Subscription, "", "Refresh", null);
-                final AzureString title = AzureOperationBundle.title("account.refresh_subscriptions");
+                final AzureString title = AzureMessageBundle.message("account.refresh_subscriptions");
```

Comment:
you should use `AzureOperationBundle.title(xx)` since this is actually an operation title. and move the message property to `operation.properties`.